server/src/server.js

```diff
@@ -151,6 +160,12 @@ app.get("/clip/:id/:domain/:clipId", function (req, res) {
       return;
     }
     let image = clip.imageBinary();
+    if (req.userAnalytics) {
+      req.userAnalytics.pageview(`/clip/${req.params.id}/${req.params.domain}/${req.params.clipId}`).send();
```

Comment:
While I don't think there's a real security issue here, on general principle these should be URL-quoted (i.e., `${encodeURIComponent(req.params.id)}`)